## 04-remove-jsmin-dfsg.patch by Christian Welzel <gawain@camlann.de>
##
## DP: remove invocation code of jsmin, which is not dfsg compliant
## DP: and therefore removed during packaging.

--- typo3.orig/t3lib/class.t3lib_div.php
+++ typo3/t3lib/class.t3lib_div.php
@@ -2474,16 +2474,17 @@
 	 * @return string Minified script or source string if error happened
 	 */
 	public static function minifyJavaScript($script, &$error = '') {
-		require_once(PATH_typo3 . 'contrib/jsmin/jsmin.php');
-		try {
-			$error = '';
-			$script = trim(JSMin::minify(str_replace(CR, '', $script)));
-		}
-		catch (JSMinException $e) {
-			$error = 'Error while minifying JavaScript: ' . $e->getMessage();
-			self::devLog($error, 't3lib_div', 2,
-				array('JavaScript' => $script, 'Stack trace' => $e->getTrace()));
-		}
+#		require_once(PATH_typo3 . 'contrib/jsmin/jsmin.php');
+#		try {
+#			$error = '';
+#			$script = trim(JSMin::minify(str_replace(CR, '', $script)));
+			$script = trim($script);
+#		}
+#		catch (JSMinException $e) {
+#			$error = 'Error while minifying JavaScript: ' . $e->getMessage();
+#			self::devLog($error, 't3lib_div', 2,
+#				array('JavaScript' => $script, 'Stack trace' => $e->getTrace()));
+#		}
 		return $script;
 	}
 
